packages/cozy-ccc-libs/README.md

```diff
@@ -4,17 +4,17 @@ side.
 To get the libs necessary on content script side use :
 
 ```javascript
-import libs from 'cozy-ccc-libs/contentscript'
+import libs from 'cozy-ccc-libs/dist/contentscript'
 ```
 
 To get the libs necessary in the launcher use :
 
 ```javascript
-import launcherLibs from 'cozy-ccc-libs/launcher'
+import launcherLibs from 'cozy-ccc-libs/dist/launcher'
 ```
 
 To get the bridge necessary to allow communication between content script and launcher use :
 
 ```javascript
-import bridge from 'cozy-ccc-libs/bridge'
+import bridge from 'cozy-ccc-libs/dist/bridge'
```

Comment:
Why do we impose the `dist` folder? Can't we add `dist` to the entry point in the package.json ?